### DISCUSSION TITLE
Move file or folder to different folder

### DISCUSSION THREAD

**Discussion by nathanl** [damionx7/Zettel-Notes-Documentation#158] [Feature Requests]
I like to organize my notes in folders. Currently I see a way to create a new folder in the current location, but I don't see a way to move an existing file or folder to a different parent folder.

**Comment by damionx7** [comment]
Hey. This feature already exists. You can long press note / folder and choose to move or copy the folder.

**Reply by nathanl** [reply]
Ah, OK, thank you, now I see! So the ability to move things is awesome, but I think the labeling could be better.

- The "three dot" menu on the file has "Copy/Move", but pressing that gives the options "move Note to repository" and "copy Note to repository", nothing about folders.  If I tap that, it lists repositories and if I tap a repository name, it goes to the top folder of that repository. I had discovered this much but didn't notice the arrow on the right that I can tap to choose a folder within that repository. So if this were labelled "move Note to folder in repository" I think that would give a better hint. Possibly an intermediate step where you tap a repository to move to and then choose "top-level" or "folder" would be clearer than tapping on the left vs right side of an item.
- On long press, the scissors are labelled "move Note to repository", but actually just moves between folders in the same repository as I wanted - I think this should be labelled "move Note to folder"

**Reply by damionx7** [reply]
Actually, when you press back in the dialog which is shown after clicking move / copy note to repository, then list of all repositories will be shown.

Initially this feature was named as move and copy but to indicate that files can be moved in between repositories, it was renamed to move / copy note to repository.
